modelserver-jsonforms-property-view/src/browser/modelserver-widget-provider.ts

```diff
@@ -56,21 +56,6 @@ export abstract class ModelserverAwareWidgetProvider extends JsonFormsPropertyVi
 
     protected abstract handleChanges(jsonFormsData: Object | undefined): void;
 
-    /** Returns a string representing the model URI relative to the workspace root URI.
-     * @param sourceUri a non-encoded URI string
-     */
-    protected getRelativeModelUri(sourceUri: string): string {
```

Comment:
What is the compatibility impact of this deletion? It's a protected method, so in theory is used by subclasses out in the wild.

Reply:
Perhaps, yes. But adopters can still use their implementation of the transformation to a relative modelUri if they'd like to. But as everything got more robust there is not need having to transform it anymore from my POV. Also it was just a convenience method extracted from one of your other projects as far as I can remember.

Reply:
Sorry, I keep forgetting that this API hasn't reached 1.0 yet so incompatible changes aren't really a concern!